src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -335,6 +339,7 @@
   <!-- List of external files that trigger design-time builds when they are modified -->
   <ItemGroup>
     <AdditionalDesignTimeBuildInput Include="$(ProjectAssetsFile)" />   <!-- NuGet assets file -->
+    <AdditionalDesignTimeBuildInput Include="@(PotentialEditorConfigFiles)" /> <!-- Places .editorconfig files may appear -->
```

Comment:
I think the conclusion that this wasn't needed right? 

Alternatively, we have a watcher watch "AdditionalDesignTimeBuildInputs" and we don't need one to watch PotentialEditorConfigFiles.

Reply:
This is what I expected to to move our assets file watch over to.

Reply:
I removed this for now.